pydoctor/themes/base/apidocs.css

```diff
@@ -211,10 +211,6 @@ ul ul ul ul ul ul ul {
     padding: 2px;
 }
 
-/* Section header */
-.fieldTable tr td:first-child.fieldName {
-    width: max-content;
-}
```

Comment:
Not sure from where this comes from, thought.

Reply:
After reviewing the git blame and the referenced [commit diff](https://github.com/twisted/pydoctor/commit/abb4914b5b43042764b43a45f722e3de8901ac0a#diff-12fe32cd4f1603b1cc87fcbf51992c45e28b713d8b5ce440ee228b1ebded0dcdR200), I think it's Ok to remove it. It does not change anything special, the row always takes the maximum width possible anyhow.